build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -87,15 +82,23 @@ jobs:
 
   #   displayName: 'build-nupkg.ps1'
 
+  - task: PowerShell@2
+    displayName: Update metapackage version
+    inputs:
+      targetType: 'inline'
+      script: |
+        $packageVersion = '$(version).${{ parameters.prereleaseVersionTag }}'
+        Write-Host version: $packageVersion
+        [xml]$publicNuspec = Get-Content -Path $(Build.SourcesDirectory)\build\NuSpecs\Microsoft.ProjectReunion.Foundation.nuspec
+        $publicNuspec.package.metadata.version = $packageVersion
+        Set-Content -Value $publicNuspec.OuterXml $(Build.SourcesDirectory)\build\NuSpecs\Microsoft.ProjectReunion.Foundation.nuspec
+
   - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
     displayName: 'Pack Full Nuget Package'
     inputs:
       command: 'pack'
       packagesToPack: '$(Build.SourcesDirectory)\build\NuSpecs\Microsoft.ProjectReunion.Foundation.nuspec'
-      versioningScheme: 'byPrereleaseNumber'
-      majorVersion: ${{ parameters.major }}
-      minorVersion: ${{ parameters.minor }}
-      patchVersion: ${{ parameters.patch }}
+      versioningScheme: 'off'
```

Comment:
What's this do? (sounds like a proper noun fronting a bunch of machinery)

Reply:
This is so the task doesn't use it's own versioning scheme.